### PR TITLE
docs: edit topic titles in ToC and move content contribution section in ToC

### PR DESCRIPTION
**Pull request by bob-watson**
* set all titles in left nav to sentence case
* move documentation contribution guide to top-level of ToC from inside the Reference topics
* update overview and reviewing topics to accommodate new ToC organization

## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

The documentation contribution guidelines were in the Reference section, but they really aren't reference topics. They're more procedural and would benefit from better visibility in the ToC.

Issue Number: N/A

## What is the new behavior?

The documentation contribution guidelines are now titled more clearly and visible in the top-level of the documentation's left nav.

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: aio/content/guide/contributors-guide-overview.md [hunk 1]
```diff
@@ -1,14 +1,38 @@
...
+These steps set up your local computer to contribute documentation content:
 
-## Contributing to Angular documentation
+1. Log in to your account on GitHub.
+    If you don't have an account on GitHub, [create a new account](https://github.com/join) before you continue.
+2. In your browser, navigate to `https://github.com/angular/angular`.
+3. In your browser, fork the `angular/angular` repo into your GitHub account.
+   1. From `https://github.com/angular/angular` in the browser, choose the **Fork** button near the top-right corner of the page.
+   2. If asked, choose to create a fork to contribute to the original project.
+4. On the computer that you want to edit the , clone your fork of the `angular` repo.
```
Comment (jessicajaniuk) [inline aio/content/guide/contributors-guide-overview.md:19]: There's a comma in the middle of this sentence that I think is supposed to be `code`, `project`, or `repository`.

File: aio/content/guide/contributors-guide-overview.md [hunk 2]
```diff
@@ -1,14 +1,38 @@
...
+5. In the `angular/aio` directory of your cloned repo, build and view the documentation locally.
+   1. Run these developer tasks described in the [Angular documentation README](https://github.com/angular/angular/tree/main/aio#angular-documentation-project-httpsangulario):
+      1. `yarn`
```
Comment (jessicajaniuk) [inline aio/content/guide/contributors-guide-overview.md:22]: Sometimes there are issues with the initial yarn installation. We should call out that if they have any errors with the yarn install, just re-run yarn. That should resolve the installation issues.

**Review by jessicajaniuk — APPROVED** [review]
This looks good to me other than the one line with the comma that needs to be fixed. 

This is non-blocking, but it would be nice if we had some info in the contributors guide on commit messages, keeping commits to only the relevant minimum, and squashing unnecessary commits before putting up a PR. We could also link to guides on how to use interactive rebase appropriately. People seem to really struggle when we ask them to squash commits. I'd be happy to add this in a separate PR, too.

**Comment by bob-watson** [comment]
@jessicajaniuk Thanks for the feedback. 
This is a first step in making content contributions friendly. The more I dig into this, the more work i see it needs, but I wanted to get the ball rolling by bringing this section of content into the light.

File: aio/content/guide/contributors-guide-overview.md [hunk 3]
```diff
@@ -1,14 +1,69 @@
...
+Input from you helps both new and experienced Angular developers.
 
-## Before you begin
+## Before you start
 
-Before you get started with your contributions, review [Contributing to Angular](https://github.com/angular/angular/blob/main/CONTRIBUTING.md). That topic explains many of the tasks and guidelines you need to know before you make your first pull request.
+The following steps set up your local computer to contribute documentation content.
 
-## Contributing to Angular documentation
+1.  Sign into your account on GitHub.
```
Comment (mgechev) [inline aio/content/guide/contributors-guide-overview.md:14]: We can also explain the simple path. Clicking on the edit button enables folks to easily fix typos.

**Review by mgechev — APPROVED** [review]
LGTM. My only suggestion is to consider including the simple path for making small fixes via the GitHub UI in the contributor guide.

**Comment by pkozlowski-opensource** [comment]
This PR was merged into the repository by commit 757684b705c3efc8c84db73dc3622fc24699c5a1.

**Pull request closed without merging**